settings.example.json

```diff
@@ -1,4 +1,23 @@
 {
-  "token":  "DISCORD_API_TOKEN",
-  "logFilePath": "C:\\Program Files (x86)\\Sony\\EverQuest\\Logs\\eqlog_Auclog_P1999Green.txt"
+  "logFilePath": {
+    "BLUE": "C:\\Program Files (x86)\\Sony\\EverQuest\\Logs\\eqlog_CHARNAME_project1999.txt",
```

Comment:
Blue has a slightly different ending to log file

Reply:
ah, k